source/includes/_api_keys.md

```diff
@@ -0,0 +1,302 @@
+# API keys
+
+API keys are randomly generated passwords that can be used as an authentication method to access a protected API endpoint as an authorized user.
```

Comment:
authorized and authenticated, technically

Reply:
What do you mean? The API key is an authentication method to ensure the user has the required authorization to access the endpoint, no?